common/shlibs

```diff
@@ -2470,7 +2470,7 @@ libu2f-host.so.0 libu2f-host-1.0.0_1
 libu2f-server.so.0 libu2f-server-1.0.1_2
 libsqlcipher.so.0 sqlcipher-3.3.1_1
 libgta.so.1 libgta-1.2.0_1
-libgdal.so.26 libgdal-3.0.0_1
+libgdal.so.27 libgdal-3.1.3_1
```

Comment:
This will break all of the packages listed in `xrevshlib libgdal`, they need to be revbumped to be rebuilt with the new library.

Reply:
ok

Reply:
they will all be ready by tomorrow....:)